### PR TITLE
Fixed Deprecated github repo link

### PR DESCRIPTION
**Pull request by abdulmalik97**
This link is deprecated - https://github.com/danfinlay/js-eth-personal-sign-examples (As mentioned in the repo readme)

New reference repository - https://github.com/MetaMask/test-dapp

### CODE REVIEW

**Review by FrederikBolding — APPROVED** [review]

File: docs/guide/signing-data.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Since MetaMask makes cryptographic keys available to each user, websites can use
 
 ## Signing Data with MetaMask
 
-If you’d like to jump to some working signature examples, [you can visit this repository](https://github.com/danfinlay/js-eth-personal-sign-examples).
+If you’d like to jump to some working signature examples, [you can visit this repository](https://github.com/MetaMask/test-dapp).
```
Comment (kingznqueenz) [inline docs/guide/signing-data.md:11]: I'm having technical problems with my account

**Comment by kingznqueenz** [comment]
I need a new file

File: docs/guide/signing-data.md (hunk 1 above)
Comment (kingznqueenz) [reply docs/guide/signing-data.md:11]: Will u be able to add me on again, I just took long to comment

Comment (kingznqueenz) [reply docs/guide/signing-data.md:11]: > I'm having technical problems with my account 

Checked

Comment (kingznqueenz) [reply docs/guide/signing-data.md:11]: Thank you

Comment (Gudahtt) [reply docs/guide/signing-data.md:11]: Please do not comment on PRs about unrelated topics. If you are looking for support, see: https://metamask.zendesk.com/hc/en-us

**Pull request merged**